torch/nn/quantized/functional.py

```diff
@@ -77,7 +77,7 @@ def conv2d(input, weight, bias,
     Args:
         input: quantized input tensor of shape :math:`(\text{minibatch} , \text{in\_channels} , iH , iW)`
         weight: quantized filters of shape :math:`(\text{out\_channels} , \frac{\text{in\_channels}}{\text{groups}} , kH , kW)`
-        bias: **non-quantized** bias tensor of shape :math:`(\text{out\_channels})`. The tensor type must be `torch.int32`.
+        bias: **non-quantized** bias tensor of shape :math:`(\text{out\_channels})`. The tensor type must be `torch.float`.
```

Comment:
@supriyar - is qnnpack also expecting fp32 bias? I think it's best to keep them the same. (fp32 itself sounds fine, we can re-quantize inside pack)

If they are different, given that quantization of bias is cheap, we could allow this function to receive either quantized or non-quantized bias and do conversion internally if needed.

Reply:
qnnpack expects int32 bias. We prepack weight and bias together in qnnpack 
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/qnnpack/src/qnnpack/pack.h#L81

We decided to not change qnnpack (in the interest of time) to decouple this and instead pack bias in the fbgemm struct to make them semantically the same.
Given that I think 2nd option proposed by @dzhulgakov makes sense - i.e. change the function to accept both quantized and non-quantized bias.